dedupe/clustering.py

```diff
@@ -228,7 +229,7 @@ def cluster(dupes: numpy.ndarray,
 
```

Comment:
great observation @jhendr, let's move the created of squared_distances down here, closer to where use it.

Reply:
👍  Done